Playmate/View Controllers/Session Details View/PlayerProfileViewController.m

```diff
@@ -86,7 +93,6 @@ -(void)manageFriendButtonUI {
             self.isMyFriend = NO;
             [self setRequestPendingAsState];
         }
-        return;
```

Comment:
why did you remove the return? 
It was a bug?

Reply:
Yep, the return caused a bug since there is a corner case that was not accounted for in this if statement, so I had to allow it to continue. The purpose was for detecting whether the player's profile being viewed is friends with the logged in users.